services/qbittorrent/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner. 
```

Comment:
The template text is still here, please remove

Reply:
Fixed